contribs/src/main/java/com/netflix/conductor/contribs/PublisherModule.java

```diff
@@ -19,17 +19,20 @@
 package com.netflix.conductor.contribs;
 
 import com.google.inject.AbstractModule;
-import com.netflix.conductor.contribs.publisher.TaskStatusPublisher;
-import com.netflix.conductor.contribs.publisher.WorkflowStatusPublisher;
+import com.netflix.conductor.contribs.publisher.*;
 import com.netflix.conductor.core.execution.TaskStatusListener;
 import com.netflix.conductor.core.execution.WorkflowStatusListener;
 
 public class PublisherModule extends AbstractModule {
 
 	@Override
 	protected void configure() {
+		PublisherConfiguration configuration = new SystemPropertiesPublisherConfiguration();
+
+		bind(PublisherConfiguration.class).to(SystemPropertiesPublisherConfiguration.class);
 		bind(WorkflowStatusListener.class).to(WorkflowStatusPublisher.class);
 		bind(TaskStatusListener.class).to(TaskStatusPublisher.class);
-	}
 
+		new RestClientManager(configuration);
```

Comment:
If Guava modules have a proper close/dispose calls, we need to call the close on PoolingHttpClientConnectionManager used inside RestClientManager.